torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -787,9 +787,15 @@ def aot_dispatch_autograd(
     )
 
     fw_metadata.deterministic = torch.are_deterministic_algorithms_enabled()
-    fx_g, joint_inputs, maybe_subclass_meta = aot_dispatch_autograd_graph(
-        flat_fn, flat_args, aot_config, fw_metadata=fw_metadata
-    )
+    with dynamo_timed(
```

Comment:
One callout is that this will give us stats for AOTDispatcher tracing in the training path, but not the inference path. If you are ok with fixing both locations in this PR so we care consistent, the other spot for it is here: https://github.com/pytorch/pytorch/blob/main/torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py#L159